core/src/main/java/greencity/service/impl/EcoNewsCommentServiceImpl.java

```diff
@@ -68,7 +68,8 @@ public AddEcoNewsCommentDtoResponse save(Long econewsId, AddEcoNewsCommentDtoReq
      */
     @Override
     public PageableDto<EcoNewsCommentDto> findAllComments(Pageable pageable, User user, Long ecoNewsId) {
-        Page<EcoNewsComment> pages = ecoNewsCommentRepo.findAllByParentCommentIsNullAndEcoNewsIdOrderByCreatedDateDesc(
+        ecoNewsService.findById(ecoNewsId);
```

Comment:
Why does `findById` is called and the result of the call is ignored?

Reply:
It only checks whether econews with this id exists